gengokit/handler/handler_test.go

```diff
@@ -1,29 +1,34 @@
 package handler
 
 import (
-	log "github.com/Sirupsen/logrus"
 	"go/ast"
 	"go/parser"
 	"go/token"
+	"io"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 	"strings"
 	"testing"
 
+	log "github.com/Sirupsen/logrus"
+	"github.com/pkg/errors"
+
 	"github.com/TuneLab/go-truss/gengokit"
-	thelper "github.com/TuneLab/go-truss/gengokit/gentesthelper"
+	helper "github.com/TuneLab/go-truss/gengokit/gentesthelper"
 	"github.com/TuneLab/go-truss/svcdef"
 )
 
 var gopath []string
+var diff = helper.DiffStrings
+var testFormat = helper.TestFormat
 
 func init() {
 	gopath = filepath.SplitList(os.Getenv("GOPATH"))
 }
 
 func init() {
-	_ = thelper.DiffStrings
+	_ = helper.DiffStrings
```

Comment:
Why is this in here? The package is being used. This line should probably be deleted.